nsis/src/nsis/file/NsisExecutable.java

```diff
@@ -29,28 +34,54 @@ public static NsisExecutable createNsisExecutable(GenericFactory factory,
 	private void initNsisExecutable(GenericFactory factory, ByteProvider bp,
 			SectionLayout layout) throws IOException {
 		this.reader = new FactoryBundledWithBinaryReader(factory, bp, true);
-		this.header_offset = -1;
+		this.headerOffset = -1;
```

Comment:
Looks good, I had meant to send this if you have not already seen it: https://google.github.io/styleguide/javaguide.html

If you're unsure of something follow that.

Reply:
Thanks! That is exactly how I caught that style error